client/src/css/index.css

```diff
@@ -569,7 +570,12 @@ p {
 }
 
 .place a:hover { color: #aaa; }
-.place .link-text { border-color: #ddd; }
+.place .link-text {
+  border-color: #ddd;
+  letter-spacing: 0.05ex;
+  font-size: 0.96em;
```

Comment:
I like the letter-spacing and line-height, but I'm not so sure about the slight font-size decrease. Any reason to do this inside .link-text, or really at all? The current font size looks fine to me.